requirements.txt

```diff
@@ -4,4 +4,4 @@ Flask
 Shapely
 Flask_Cors
 PyYAML
-geojson
+json
```

Comment:
`json` ships with Python and thus doesn't need to be specified in `requirements.txt`.